telegram/utils/helpers.py

```diff
@@ -72,3 +73,45 @@ def from_timestamp(unixtime):
         return None
 
     return datetime.fromtimestamp(unixtime)
+
+
+def mention(user, html=False, name=None):
+    """
+    Args:
+        user (:obj:`int` | :class:`telegram.User` | :class:`telegram.Message`)
+            The user's id, or User object, or a Message from the user which you want to mention.
+        html (:obj:`bool`) Default output as markdown, pass True to use HTML.
+        name (:obj:`str`) The name the mention is showing. Required when id is provided in user.
+            Optional if User or Message is provided and will overwrite the name in user.
+
+    Returns:
+        str:
+    """
+    if not user:
+        return None
+
+    if isinstance(user, Message):
+        user = Message.from_user
+
+    if isinstance(user, int) and name:
+        if html:
+            return '<a href="tg://user?id={}">{}</a>'.format(user, escape_html(name))
+        else:
+            return "[{}](tg://user?id={}".format(escape_markdown(name), user)
+
+    elif isinstance(user, User):
+        if not name:
+            if user.last_name:
```

Comment:
The user object already has a `name`-property, wouldn;t it make more sense to use that?
https://github.com/python-telegram-bot/python-telegram-bot/blob/master/telegram/user.py#L67